deploy/kubernetes/base/controller.yaml

```diff
@@ -41,13 +41,12 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: spec.nodeName
-            - name: IBMCLOUD_API_KEY
-              valueFrom:
-                secretKeyRef:
-                  name: ibm-secret
-                  key: IBMCLOUD_API_KEY
-                  optional: true
+            - name: API_KEY_PATH
+              value: /var/secrets/IBMCLOUD_API_KEY
```

Comment:
is `/var` a standard path for the credential?

Reply:
As far as I know, there's no one standard path as such, it could be /etc, /var, /mnt.

Reply:
/etc looks to be better place..